bin/create-db-seed

```diff
@@ -17,4 +17,4 @@ fi
 source "${localSettings}"
 
 # Create new database seed
-mysqldump --skip-extended-insert --order-by-primary ${dbauth} ${dbhost} "${dbname}" >"$(readlink -f $(dirname $(dirname ${0})))/db/seed.sql"
+mysqldump --hex-blob --skip-extended-insert --order-by-primary ${dbauth} ${dbhost} "${dbname}" >"$(readlink -f $(dirname $(dirname ${0})))/db/seed.sql"
```

Comment:
Otherwise the images were output in binary which was not fun for the utf-8 encoding.